addons/account/controllers/portal.py

```diff
@@ -14,9 +14,13 @@ class PortalAccount(CustomerPortal):
     def _prepare_home_portal_values(self, counters):
         values = super()._prepare_home_portal_values(counters)
         if 'invoice_count' in counters:
-            invoice_count = request.env['account.move'].search_count(self._get_invoices_domain()) \
+            invoice_count = request.env['account.move'].search_count([('move_type', 'in', ('out_invoice', 'out_refund'))] + self._get_invoices_domain()) \
```

Comment:
Why are we not including the receipts anymore?

Reply:
It seems to be an oversight from my part, will fix it. 

Functionally, I don't exactly know what they rerpresent, but i guess out_receipt are linked to invoice and in_receipt to bills ?

Reply:
It's basically invoices without a mandatory partner, and you are right for the types: `out` says with `out` and `in` stays with `in`